src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -180,6 +180,8 @@
 	private final JSpinner particleSpawningRadious = new JSpinner(new SpinnerNumberModel(0.5, 0, 2, 0.1f));
 	private final JSpinner particleAmount = new JSpinner(new SpinnerNumberModel(4, 0, 1000, 1));
 	private final JSpinner slipperiness = new JSpinner(new SpinnerNumberModel(0.6, 0.01, 5, 0.1));
+	private final JSpinner speedFactor = new JSpinner(new SpinnerNumberModel(1, 0.01, 1000, 0.1));

```

Comment:
I'd change the minimal value of `speedFactor` to 0 (it could be negative, but it's wonky), and the minimal value of `jumpFactor` to -1000 (doesn't do anything weird, it only negates the jump boost effect). Also applies to plant GUI spinners.

Reply:
Is this what the default value in the game is? Maybe if there is no default value, code may be omitted in such cases.

Reply:
I checked the default values in the code and they are set to 1.0.